contrib/in-memory-accumulo/src/main/java/datawave/accumulo/inmemory/InMemoryBatchScanner.java

```diff
@@ -69,6 +88,27 @@ public void setRanges(Collection<Range> ranges) {
         }
         return chain;
     }
+
+    @Override
+    public Iterator<Entry<Key,Value>> rebuild(Key lastKey) {
+        // rebuild the ranges
+        List<Range> newRanges = new ArrayList<>();
+        boolean found = false;
+        for (Range range : ranges) {
+            if (found) {
+                newRanges.add(range);
+            } else if (range.contains(lastKey)) {
```

Comment:
It is not clear to me what the purpose of the rebuild as the interface does not have any docs. As the code exists it includes all ranges starting with the lastKey (inclusive). I believe this is the intended result based upon the brief discussion we had concerning the expected  behavior.

Reply:
I will add documentation as the code is not as obvious as I had thought.

Reply:
done